test/Microsoft.Azure.WebJobs.Logging.FunctionalTests/LoggerTest.cs

```diff
@@ -37,7 +37,23 @@ public void Dispose()
                 tasks.Add(table.DeleteIfExistsAsync());
             }
 
-            Task.WaitAll(tasks.ToArray());
+            try
+            {
+                Task.WaitAll(tasks.ToArray());
+            }
+            catch (AggregateException aex)
+            {
+                if (!aex.InnerExceptions.All(e =>
```

Comment:
nit: use `.Handle()` instead

Reply:
I somehow didn't know that existed. Sweet.